From 38b46a98790d59eff3a17ac679f0f1bd77c0db2d Mon Sep 17 00:00:00 2001
From: Jayanthi A <jayanthi.annadurai@cavium.com>
Date: Thu, 4 Mar 2021 10:57:08 -0800
Subject: [PATCH 1287/1921] devicetree: bindings: Update Cadence SD driver
 parameter

Update driver compatible parameter for Cadence SD host controller
version 6.

Change-Id: I25baa326e0799fbec14aeb8ebfee0274561674c6
Signed-off-by: Jayanthi A <jayanthi.annadurai@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/47212
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 Documentation/devicetree/bindings/mmc/sdhci-cadence.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/mmc/sdhci-cadence.txt b/Documentation/devicetree/bindings/mmc/sdhci-cadence.txt
index fa423c277853..8075a62038b8 100644
--- a/Documentation/devicetree/bindings/mmc/sdhci-cadence.txt
+++ b/Documentation/devicetree/bindings/mmc/sdhci-cadence.txt
@@ -3,6 +3,7 @@
 Required properties:
 - compatible: should be one of the following:
     "cdns,sd4hc"               - default of the IP
+    "cdns,sd6hc"               - for Cadence SD host controller version 6
     "socionext,uniphier-sd4hc" - for Socionext UniPhier SoCs
 - reg: offset and length of the register set for the device.
 - interrupts: a single interrupt specifier.
-- 
2.31.1

